app/models/test_passage.rb

```diff
@@ -48,6 +49,15 @@ def next_question
     end
   end
 
+  def timer_valid?
+    second_to_end.positive?
+  end
+
+  def second_to_end
```

Comment:
Предложение по неймингу
```suggestion
  def time_left
```